WFP-VAM/prism-app
# Feature/admin level post requests

This fixes issue https://github.com/WFP-VAM/prism-app/issues/722.

How to test the feature:
- country=mongolia
- activate 'capacity/hay harvest' layer
- see that a post request is made

This is work in progress
